NewYearGift/NewYearGift/Models/Sweet.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewYearGift.Models
+{
+    interface IShowSweet
+    {
+        void ShowSweet()
+        {
+            
+        }
+    }
+   
+    internal class Sweet : IShowSweet
+    {
+        public int length;
+        public int height;
+        public int width;
+        public int volume;
+        public double density;
+        public double mass;
+        public string[] names = new string[10] { "Snikers", "Mars", "Bounty", "Roshen", "AVK", "Konti", "Milky Way", "Twix", "Milka", "Nestle" };
+        public string name;
+        
+        public Sweet()
+        {
+            length = new Random().Next(1, 8);
+            height = new Random().Next(1, 6);
+            width = new Random().Next(1, 4);
+            volume = length * height * width;
+            density = new Random().NextDouble();
+            mass = density * volume;
+            name = names[new Random().Next(0, 10)];
+        }
+
+        public void ShowSweet()
+        {
+            Console.WriteLine("This is {0} sweet", name);
+        }
+
```

Comment:
This method should be in SweetService and exectly this class should realize IShowSweet